src/main/java/org/mskcc/limsrest/service/CustomForm.java

```diff
@@ -0,0 +1,54 @@
+package org.mskcc.limsrest.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomForm {
+    private String id;
+    private String name;
+    private String note;
+    private Map<String, String> fields;
+
+    public CustomForm() {
```

Comment:
Add class comment to describe how it is used

Reply:
Addressed.